src/css/config.css

```
@@ -133,20 +180,23 @@
-    background-color: #ddf;
+    background-color: #262626;
+    color: #eee;
 }
 
 .axpc_config_table td {
     padding: 0 8px;
-    border: 1px solid #aaa;
+    border: 1px solid #333;
 }
 
 /* 設定用のカラーパレット表示（モックアップ）*/
```

Stylelint (17.14.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)